lib/plugins/TaskRunner/LAMPApp.js

```diff
@@ -105,12 +113,19 @@ class LAMPApp extends Script {
     ]);
   }
 
-  addScriptCreateDatbase() {
+  addScriptCreateDatabase() {
     this.script = this.script.concat([
       'echo \'Creating MySQL Database, user and granting access.\'',
-      'mysql -e \'create database ' + this.databaseName + '\'',
-      'mysql -e \'grant all on ' + this.databaseName + '.* to "root"@"localhost"\'',
+      'set +ux', // Don't post variables to build logs.
+      'DATABASE_NAME='+this.databaseName,
+      'DATABASE_USER='+this.databaseUser,
+      'DATABASE_PASS='+this.databasePass,
+      'if [ -e $ASSET_DIR/credentials.sh ]; then source $ASSET_DIR/credentials.sh; fi', // Set db credentials if found.
+      'mysql -e \'create database \'$DATABASE_NAME',
+      'if [ "$DATABASE_USER" != "root" ]; then mysql -e \'create user "\'$DATABASE_USER\'"@"localhost" identified by "\'$DATABASE_PASS\'"\'; fi',
```

Comment:
Kind of weird that we had the option for DatabaseUser but no command to actually create that user.